package.json

```diff
@@ -1,49 +1,44 @@
 {
-  "name": "bondage",
-  "version": "2.2.0",
+  "name": "@mnbroatch/bondage",
+  "version": "4.0.1",
   "description": "Yarn parser for Javascript",
-  "main": "src/bondage.js",
+  "main": "dist/bondage.js",
```

Comment:
If people want to transpile for themselves, they can use the `src/index` entrypoint. Otherwise, this and its minified counterpart are UMD modules that can be used in node or in a browser.